cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CertUtils.java

```diff
@@ -59,6 +59,19 @@ public static String getCertificateThumbprint(Secret certSecret, String key) {
         }
     }
 
+    /**
+     * Generates the expiration date as epoch of the server certificate which is used to track when the certificate changes and rolling update needs to be triggered.
+     *
+     * @param certSecret    Secrets with the certificate
+     * @param key           Key under which the certificate is stored in the Secret
+     * @return              Epoch representation of the expiration date of the certificate or 0 if certSecret contains no valid X509Certificate
+     * @throws              RuntimeException if the certificate cannot be decoded
+     */
+    public static long getCertificateExpirationDateEpoch(Secret certSecret, String key) {
```

Comment:
Would it make sense to have this part of the `Ca` class?

Reply:
I guess it does yes, maybe `getCertificateThumbprint` and `getCertificateShortThumbprint` should be moved as well, as `getCertificateExpirationDateEpoch` is done in those fashions.

Reply:
I think the difference is whether you pass there random certificate (which I suspect you do for the other methods) or just always the CA (which is what I suspect you do here). So I'm not sure they way the other methods are used is identical.